### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by vchrombie**
Pull Request for the Issue #7 
- Added badges according to the project.

- Added the Installation steps.

- Added the demo pictures for the installation.

- Added some information about the extension.

- Added a lot of emojis. :heart: 

Can you please review this PR?

### CODE REVIEW

**Comment by vchrombie** [comment]
Can you review my PR #8 please.

I have doubt regarding the issue #6.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+ Click on the logo and then `Show`
+
+![image](https://raw.githubusercontent.com/vchrombie/YoutubeDataCollector/52968873445d24d11fa85f5488f309dc110449e8/img/demo5.png)
+
+It displays the recent youtube videos opened and displays them in a list form. :raised_hands:
+
+![image](https://github.com/vchrombie/YoutubeDataCollector/blob/master/img/demo6.png?raw=true)
+
+
+**Found a :bug:  bug?** [Open an issue](https://github.com/Parth-Vader/YoutubeDataCollector/issues/new) with some [emojis](http://emoji.muan.co). Issues without emojis are not valid. :stuck_out_tongue_winking_eye:
```
Comment (Parth-Vader) [inline README.md]: `Issues without emojis are not valid` can be removed. It's not needed.

Comment (vchrombie) [reply README.md]: Ok.
I will remove it.

**Comment by vchrombie** [comment]
I have done that.
Can you review it now?

**Comment by Parth-Vader** [comment]
Looks good to me, but the images used would have to be updated once the logo is finalized, and other changes are made. Merging it now, but please try to update them as changes take place.

**Comment by vchrombie** [comment]
Thank You.
I will take care of that. :)

**Pull request merged**